.github/workflows/example-basic.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         # because we are running these examples in a monorepo
         with:
           working-directory: examples/basic
+          config-file: cypress.json
```

Comment:
This is part of a much larger discussion, but mine as well bring it up now. All of the examples in this repo are for Cypress < 10. So the easiest way to get all of these to pass was by adding `config-file: cypress.json` to all of them where necessary.

Should we, at some point, update all of these examples to v10? Or do we have examples for < 10 and the same examples updated to work with v10?